build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -144,17 +144,11 @@
 
 # Batched Tensorflow models.
 BERT_LARGE_TF_BATCHES = [
-    model
-    for batch_size, model in tf_models.BERT_LARGE_384_FP32_TF_BATCHES.items()
-    # Higher batch sizes disabled due to OOM https://github.com/openxla/iree/issues/14668.
-    if batch_size < 64
+    model for batch_size, model in tf_models.BERT_LARGE_384_FP32_TF_BATCHES.items()
```

Comment:
Regenerate CMake files? This PR should look like a revert of https://github.com/openxla/iree/commit/80bb1e0607c7de12daa468a2a42f0dde6e1c5ce1, yeah?

Reply:
I ran that it is strangely not generating any new CMake lines. Will debug.